src/Header.js

```diff
@@ -1,11 +1,36 @@
 import { Component } from 'react';
 import Navbar from 'react-bootstrap/Navbar'
 import { Container } from 'react-bootstrap';
+import Form from 'react-bootstrap/Form'
+import BeastData from './data.json'
+import ListGroup from 'react-bootstrap/ListGroup'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+
+
+
+
 class Header extends Component{
+ 
+  handleEvent=(e)=>{
+   let selection =e.target.value;
+   this.props.beastData(selection);
+  };
+  
```

Comment:
Hi there!

So in Canvas you asked about how to update data for search. You have part of the pieces here. 

You will need to set the state of filtered search to the contents of the form field you want to capture 👍